server/.eslintrc.js

```diff
@@ -7,8 +7,8 @@ module.exports = {
     "extends": [
         "eslint:recommended",
         "plugin:react/recommended",
-        "airbnb",
-        "airbnb/hooks"
+        // "airbnb",
+        // "airbnb/hooks"
```

Comment:
what in the world is this stuff?

Reply:
It seems to have it added a airbnb style correction guide. I wasn't crazy about all the squiggly lines but I may add it back at the very end and clean things up that I didn't do correctly. :S